src/lib.rs

```diff
@@ -185,6 +186,12 @@ impl Exporter {
         is_metric_enabled!(exporter, health);
     }
 
+    fn spawn_stats(exporter: Self) {
```

Comment:
Wouldn't it make more sense to define method definition as `fn spawn_stats(self)`?

Reply:
true, updated